### ISSUE TITLE
COMMUNITY CALL: Git Graph is abandoned— let's get VS Code to rebuild it as an official feature

### ISSUE THREAD

**Issue by Keavon** [mhutchie/vscode-git-graph#838]
tl;dr: Click the 👍 on this VS Code issue --> https://github.com/microsoft/vscode/issues/179053

---

*I'm reposting #804 for visibility since it has left the first page of issues. The last post helped garner 300+ upvotes.*

Sadly, Git Graph seems to be abandoned. Currently there are over ~~7~~ (*update:* **9**) million of us who have installed this extension, and probably millions of us use it daily as an indispensable tool in our workflows. But holding out hope for continued maintenance looks unlikely at this point. And this extension lacks an open source license, so a fork won't be possible. That means it must be reimplemented by either somebody making a competing extension, or by the VS Code team as a built-in feature.

The core idea, for a minimalistic, unopinionated Git visualization has proven to be very compelling. JetBrains IDEs have a similar panel as well. Millions of us love the straightforward UI and features of this extension.

I believe the best way forward for us, as a community, is to campaign for a Git Graph inspired panel to be included as a built-in first-class feature in VS Code. That way it could receive official support and ongoing maintenance. And it would be available to millions more people who just aren't aware of this extension. The clock is ticking before VS Code changes enough to break this extension outright ([again](https://github.com/mhutchie/vscode-git-graph/issues/794)).

[So join me in supporting this VS Code feature request](https://github.com/microsoft/vscode/issues/179053) by clicking the 👍 on the original (top) post to vote for it, and leaving comments in support of the feature. Microsoft prioritizes feature requests based on the number of thumbs up reactions on the original post. As a benchmark, the top-rated issue is 2900 votes and we'll need over 550 to move from page 2 to [page 1](https://github.com/microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) of top-voted issues. My first time posting this brought it from near 0 to over 300 upvotes. Let's try and double that to reach page 1. Share this with anyone you know who uses Git Graph.
